pkg/api/vlabs/types.go

```diff
@@ -602,6 +604,31 @@ func (m *MasterProfile) IsCoreOS() bool {
 	return m.Distro == CoreOS
 }
 
+// IsUbuntu1604 returns true if the master profile distro is based on Ubuntu 16.04
+func (m *MasterProfile) IsUbuntu1604() bool {
```

Comment:
let's add the corresponding unit tests from the api/ pkg as well to avoid decreasing our coverage on this file

https://codecov.io/gh/Azure/aks-engine/compare/77822c37674a7ec81803a2dae6d2d082a20dd5ab...75a21cb62c4ad6e9af57c9d8cd8d1151da84b7dc/src/pkg/api/vlabs/types.go